addons/web/static/src/core/emoji_picker/emoji_picker.xml

```diff
@@ -110,13 +110,13 @@
 </t>
 
 <t t-name="web.EmojiPicker.emoji">
-    <span class="o-Emoji cursor-pointer d-flex justify-content-center rounded-3 align-items-center align-self-stretch" t-att-class="{ 'o-active': state.activeEmojiIndex === itemIndex, 'fs-2': !ui.isSmall, 'fs-1': ui.isSmall }" t-att-title="emoji.name" t-att-data-codepoints="emoji.codepoints" t-att-data-index="itemIndex" t-att-data-category="inRecent ? recentCategory.sortId : categories.find(c => c.name === emoji.category).sortId" t-on-click="selectEmoji">
+    <span class="o-Emoji cursor-pointer d-flex justify-content-center rounded-3 align-items-center align-self-stretch" t-att-class="{ 'o-active': state.activeEmojiIndex === itemIndex, 'fs-2': !ui.isSmall, 'fs-1': ui.isSmall }" t-att-title="emoji.name" t-att-data-codepoints="emoji.codepoints" t-att-data-index="itemIndex" t-att-data-category="inRecent ? recentCategory.sortId : categories.find(c => c.name === emoji.category).sortId" t-on-click="selectEmoji" t-on-mouseenter="() => this.onMouseEnter(ev, emoji)" t-on-mouseleave="() => this.onMouseLeave(ev, emoji)">
         <span t-esc="emoji.codepoints"/>
     </span>
 </t>
 
 <t t-name="web.EmojiPicker.searchInput">
-    <input class="form-control border-0 flex-grow-1 rounded-3 rounded-end-0 o-active lh-1" placeholder="Search emoji" t-model="localState.searchTerm" t-ref="autofocus" t-att-model="localState.searchTerm" t-on-input="() => this.state.activeEmojiIndex = 0" t-att-tabindex="isMobileOS ? -1 : 0"/>
+    <input type="search" class="form-control border-0 flex-grow-1 rounded-3 rounded-end-0 o-active lh-1" t-att-placeholder="placeholder" t-model="localState.searchTerm" t-ref="autofocus" t-att-model="localState.searchTerm" t-on-input="() => this.state.activeEmojiIndex = 0" t-att-tabindex="isMobileOS ? -1 : 0"/>
```

Comment:
The problem with `type="search"` is that the specific features are browser-specific, and the style are also handled by user-agent. This doesn't look nice in some browsers:

![Screenshot 2025-01-28 at 11 30 35](https://github.com/user-attachments/assets/7c4675e0-879a-4577-9aed-310e3f31912c)

I would not put `type="search"`, instead use custom classname if we want to target it specifically. The additional feature and style, if we want, should be done in another PR and user-agent independent.

Reply:
I tried with:
```css
.o-EmojiPicker-search input {
   -webkit-appearance: textfield;
   -moz-appearance: textfield;
   appearance: textfield;
}
```

But this doesn't work: the "x" is still shown and stylized per browser.